src/core/gui/PageView.cpp

```diff
@@ -793,12 +793,7 @@ double XojPageView::getWidth() const { return page->getWidth(); }
 double XojPageView::getHeight() const { return page->getHeight(); }
 
 void XojPageView::rerenderRect(double x, double y, double width, double height) {
-    int rx = std::lround(std::max(x - 10, 0.0));
-    int ry = std::lround(std::max(y - 10, 0.0));
-    int rwidth = std::lround(width + 20);
-    int rheight = std::lround(height + 20);
-
-    addRerenderRect(rx, ry, rwidth, rheight);
+    addRerenderRect(x, y, width, height);
```

Comment:
I guess here, we could actually remove XojPageView::addRerenderRect entirely (it's not called anywhere else) and move its content to XojPageView::rerenderRect directly.